samza-core/src/main/java/org/apache/samza/container/LocalityManager.java

```diff
@@ -35,56 +33,25 @@
  * Locality Manager is used to persist and read the container-to-host
  * assignment information from the coordinator stream
  * */
-public class LocalityManager extends AbstractCoordinatorStreamManager {
+public class LocalityManager {
   private static final Logger log = LoggerFactory.getLogger(LocalityManager.class);
   private Map<String, Map<String, String>> containerToHostMapping = new HashMap<>();
   private final TaskAssignmentManager taskAssignmentManager;
   private final boolean writeOnly;
+  private CoordinatorStreamManager coordinatorStreamManager;
+  private static final String CONTAINER_PREFIX = "SamzaContainer-";
```

Comment:
Nitpick: Sort class fields, here and in other classes. Usual convention is statics before final before non-final, groups separated by a newline.

Reply:
Also, why does this say "SamzaContainer" instead of this class' name?

Reply:
My bad, I thought this was the "SOURCE" like the other constants. Makes sense.